examples/flax/image-captioning/run_image_captioning_flax.py

```diff
@@ -475,18 +475,18 @@ def main():
         model_args.model_name_or_path,
         seed=training_args.seed,
         dtype=getattr(jnp, model_args.dtype),
-        use_auth_token=True if model_args.use_auth_token else None,
+        token=True if model_args.use_auth_token else None,
```

Comment:
I don't touch `model_args.use_auth_token` at this moment. It will take a bit extra work, see following.
I can do it, but would like to know your opinion and/or suggestion first.


```python
class ModelArguments:
    """
    Arguments pertaining to which model/config/tokenizer we are going to fine-tune from.
    """
    token: bool = field(
        default=False,
        metadata={
            "help": (
                "Will use the token generated when running `huggingface-cli login` (necessary to use this script "
                "with private models)."
            )
        },
    )
    use_auth_token: bool = field(
        default=False,
        metadata={
            "help": ("Deprecated argument. Use `token` instead.")
        },
    )
```
and add warning etc.

Reply:
Would be nice to do as well, but it can be in a followup PR.

Reply:
OK, I will do it in a next PR. And then do it for `datasets` and pin recent versions next. (So if anything goes wrong, easier to revert or fix). Thanks for the suggestion.